trunk/NDHMS/Routing/module_RT.F

```diff
@@ -1165,6 +1165,14 @@ subroutine LandRT_ini(did)
       allocate (rt_domain(did)%z_max (rt_domain(did)%numbasns))
       allocate (rt_domain(did)%basns_area (rt_domain(did)%numbasns))
 
+      if(nlst_rt(did)%bucket_loss .eq. 1) then
+         allocate(rt_domain(did)%qloss_gwsubbas(rt_domain(did)%numbasns))
```

Comment:
do these deallocate? I prefer explicit deallocation. it could be very useful at some later point.

Reply:
They don't, but nor do the other similar arrays.